test/verify/check-kdump

```diff
@@ -583,113 +593,108 @@ class TestKdumpNFSAnsible(KdumpHelpers, packagelib.PackageCase):
 @testlib.skipOstree("kexec-tools not installed")
 @testlib.onlyImage("Only Fedora has ansible on the image", "fedora-*")
 @testlib.timeout(900)
-class TestKdumpAnsible(KdumpHelpers, packagelib.PackageCase):
+class TestKdumpAnsible(KdumpHelpers):
     provision = {
-        "ansible_machine": {"memory_mb": 900},
         "cockpit": {"memory_mb": 512},
+        "kdump_ansible_machine": {"memory_mb": 900},
     }
 
     def testBasic(self):
-        """
-        Test the automation script generated by the kdump page on a fresh
-        machine. The first machine is the pristine machine where we run ansible
-        as we need some helpers only available to self.
-        """
-
-        b = self.browser
         m = self.machine
+        b = self.browser
+
+        kdump_machine = self.machines['kdump_ansible_machine']
 
-        m_cockpit = self.machines['cockpit']
-        m_cockpit.start_cockpit()
-        b_cockpit = self.new_browser(m_cockpit)
+        # Drop rpm repos as otherwise dnf will hang when we run the ansible role
+        kdump_machine.execute("rm -rf /etc/yum.repos.d/* /var/cache/yum/* /var/cache/dnf/*")
```

Comment:
This is much better indeed!

Reply:
And confirmed that coverage is now green! https://cockpit-logs.us-east-1.linodeobjects.com/pull-19652-20231124-162946-82a8bce8-fedora-39-devel/Coverage/pkg/kdump/kdump-view.jsx.gcov.html

NFs not as I have not ported that over but that will happen on Monday